CRITICAL - 2020-10-27 02:48:04 --> Unable to connect to the database.
#0 C:\Users\USER\Desktop\project\fispedia\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\Users\USER\Desktop\project\fispedia\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\Users\USER\Desktop\project\fispedia\vendor\codeigniter4\framework\system\Model.php(506): CodeIgniter\Database\BaseBuilder->get()
#3 C:\Users\USER\Desktop\project\fispedia\app\ThirdParty\myth-auth\src\Authentication\LocalAuthenticator.php(156): CodeIgniter\Model->first()
#4 C:\Users\USER\Desktop\project\fispedia\app\ThirdParty\myth-auth\src\Authentication\LocalAuthenticator.php(20): Myth\Auth\Authentication\LocalAuthenticator->validate(Array, true)
#5 C:\Users\USER\Desktop\project\fispedia\app\ThirdParty\myth-auth\src\Controllers\AuthController.php(85): Myth\Auth\Authentication\LocalAuthenticator->attempt(Array, false)
#6 C:\Users\USER\Desktop\project\fispedia\vendor\codeigniter4\framework\system\CodeIgniter.php(918): Myth\Auth\Controllers\AuthController->attemptLogin()
#7 C:\Users\USER\Desktop\project\fispedia\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#8 C:\Users\USER\Desktop\project\fispedia\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\USER\Desktop\project\fispedia\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 C:\Users\USER\Desktop\project\fispedia\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\Users\\USER\\D...')
#11 {main}
CRITICAL - 2020-10-27 02:48:04 --> Uncaught ErrorException: print_r(): Property access is not allowed yet in C:\Users\USER\Desktop\project\fispedia\app\Views\errors\html\error_exception.php:100
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'print_r(): Prop...', 'C:\\Users\\USER\\D...', 100, Array)
#1 C:\Users\USER\Desktop\project\fispedia\app\Views\errors\html\error_exception.php(100): print_r(Object(Myth\Auth\Controllers\AuthController), true)
#2 C:\Users\USER\Desktop\project\fispedia\vendor\codeigniter4\framework\system\Debug\Exceptions.php(308): include('C:\\Users\\USER\\D...')
#3 C:\Users\USER\Desktop\project\fispedia\vendor\codeigniter4\framework\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Database\Exceptions\DatabaseException), 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DatabaseException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-10-27 23:59:23 --> Cannot declare class App\Commands\AppInfo, because the name is already in use
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
